docs/deploy/howto-add-aks-to-ps1.md

```diff
@@ -480,10 +492,19 @@ deployment:
     hostName: aks-cluster-ip.westeurope.cloudapp.azure.com
 ```
 
-#### Installing the chart
+> **NOTE**: Please note that we have used `2.7.199` as the value of `image:tag` configuration parameter
+> above. That will result in `2.7.199` version of microservices to be deployed. If you want to deploy a
```

Comment:
[...]version of microservices and edge modules

Reply:
Added.